src/commons/navigationBar/NavigationBar.tsx

```diff
@@ -304,7 +305,14 @@ const NavigationBar: React.FC = () => {
 
   return (
     <>
-      <Navbar className={classNames('NavigationBar', 'primary-navbar', Classes.DARK)}>
+      <Navbar
+        className={classNames(
+          'NavigationBar',
+          'primary-navbar',
```

Comment:
It appears that you have defined `primary-navbar` twice here

Reply:
It's not twice, but it's due to the (partial) migration. The first `'primary-navbar'` refers to the global style, the `primary-navbar` class (which still exists). The second is our scoped style from our CSS module, where I introduced a new class called `primary-navbar`.

Reply:
Alright, got it!

Reply:
The global `primary-navbar` is defined in:

https://github.com/source-academy/frontend/blob/e050242b7ced98a5155189de2922a65d96045698/src/styles/_navigationBar.scss#L51-L60

The reason we can't bring it over to CSS modules is because of the nested classes `bp4-button` and `bp4-navbar-heading` that we apply custom styles too. If we port over this `primary-navbar` to CSS modules, the child classes also get transformed, and they don't target the Blueprint buttons and navbar heading components anymore.

This is another reason we shouldn't depend on the CSS API as mentioned in #2722.

For now, I think we just keep this, but ideally we'll move out from the entire CSS API + global styles in subsequent PRs.